index.html

```diff
@@ -347,6 +229,7 @@ <h2>Contact me</h2>
         <div class="last-line">
           <img src="./img/Shape.png" alt="a-line" />
         </div>
+        <div id="modals" class = "modals"></div>
```

Comment:
- [ ] Please make sure the modal backdrop color follows the original design color.
- [ ] Kindly reduce the height of the modal image so other components can fit properly

### Your Design
![image](https://user-images.githubusercontent.com/43995830/156452247-330b9555-39cc-4324-8a9e-4a0a164b8490.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/156452416-d67a7759-f7c0-48a6-8a1e-29574cd1bc32.png)